pkg/asset/asset.go

```diff
@@ -72,6 +72,9 @@ type RuntimeFile struct {
 // directory.
 func PersistToFile(asset WritableAsset, directory string) error {
 	for _, f := range asset.Files() {
+		if f == nil {
```

Comment:
I agree we want to fix the panic for an invalid tf config. I am less sure we want to catch an error here, as it might hide the stack trace which would make this nil pointer harder to debug. Perhaps a panic is appropriate at this point in the asset store? WDYT?

Reply:
Sounds fair to me. Updated.